book/src/ephemeral-local-volumes.md

```diff
@@ -2,15 +2,44 @@
 
 ## Status
 
-Status | Min K8s Version | Max K8s Version 
+### CSI Ephemeral Inline Volumes
+
+Status | Min K8s Version | Max K8s Version
 --|--|--
 Alpha | 1.15 | 1.15
-Beta | 1.16 | -
+Beta | >=1.16 | -
+
+### Generic Ephemeral Inline Volumes
+
+Status | Min K8s Version | Max K8s Version
+--|--|--
+Alpha | 1.19 | -
 
 ## Overview
-Traditionally, volumes that are backed by CSI drivers can only be used with a `PersistentVolume` and `PersistentVolumeClaim` object combination. This feature supports ephemeral storage use cases and allows CSI volumes to be specified directly in the pod specification.  At runtime, nested inline volumes follow the ephemeral lifecycle of their associated pods where the driver handles all phases of volume operations as pods are created and destroyed.
+Traditionally, volumes that are backed by CSI drivers can only be used with a `PersistentVolume` and `PersistentVolumeClaim` object combination. The "CSI Ephemeral Inline volumes" feature allows CSI volumes to be specified directly in the pod specification for ephemeral use cases.  At runtime, nested inline volumes follow the ephemeral lifecycle of their associated pods where the driver handles all phases of volume operations as pods are created and destroyed.
```

Comment:
What if we start off the overview and introduce both options, then go into more detail into each? Something like:

"Traditionally... combination. Two different Kubernetes features allow volumes to follow the Pod's lifecycle: CSI Ephemeral volumes and Generic ephemeral volumes..."